src/common.browser/MicAudioSource.ts

```diff
@@ -78,6 +78,10 @@ export class MicAudioSource implements IAudioSource {
         return PromiseHelper.fromResult(MicAudioSource.AUDIOFORMAT);
     }
 
+    public get blob(): Promise<Blob> {
+        throw new Error("Mic input has no blob attached");
```

Comment:
>Error [](start = 18, length = 5)

Why not reject the promise? Not sure we should mix error styles.

Reply:
Changed to reject promise